interface/patient_file/education.php

```diff
@@ -111,8 +111,8 @@
     <?php Header::setupHeader(); ?>
```

Comment:
@bradymiller @stephenwaite How do I trigger this file?

Reply:
`Miscellaneous->Patient Education`

is Pickles picture a perm addition to demo data lol?
![Screenshot from 2020-08-24 16-11-04](https://user-images.githubusercontent.com/5618372/91091409-87d02e00-e624-11ea-9466-236119eba60d.png)

Reply:
nice! :)
at some point, should be able to support docs in demo data (and get pickles in there)

Reply:
that's almost like `priority blocking`